src/ai/configuration.hpp

```diff
@@ -99,8 +99,7 @@ class configuration {
 
 
 	/**
-	 * @param[in] cfg the config to be read
-	 * @param[out] parsed_cfg parsed config
+	 * @param[in,out] cfg [in] the config to be read [out] parsed config
```

Comment:
Looking at the function prototype, my suspicion is that this change is wrong and the true change is that `cfg` became `original_cfg` and `parsed_cfg` became `cfg`. Does that make sense? I'm not entirely sure though without looking at the implementation of this function.